spec/meilisearch/client/dumps_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+RSpec.describe 'MeiliSearch::Client - Dumps' do
+  before(:all) do
+    @client = MeiliSearch::Client.new($URL, $MASTER_KEY)
+    clear_all_indexes(@client)
+  end
+
+  it 'creates a new dump' do
+    response = @client.create_dump
+    expect(response).to be_a(Hash)
+    expect(response['uid']).to_not be_nil
+    expect(response['status']).to_not be_nil
+    expect(response['status']).to eq('in_progress')
```

Comment:
```suggestion
```

Reply:
I would rather remove this too accurate test because if the dump is fast to be created, this test would fail, but our package would still be correct. Checking this is not nil is enough 🙂

Reply:
@curquiza I am not sure this is true. The response to a create dump call will be `in_progress` even if the index is empty, you are working in your local machine and no matter how fast it can create it. I think this is the response MeiliSearch send in any case when you ask for a dump creation and there were no errors or conflict. You can get a `done` status when you specifically do a call on get status.

Reply:
In any case, maybe we can meet in the middle groud. We don't need to check for that specific answer, but check that the status code is not an error (4XX - 5XX)

Reply:
> I think this is the response MeiliSearch send in any case when you ask for a dump creation and there were no errors or conflict.

Ah yes you're right! It's ok for me then!

Reply:
Well, the test just failed...

Reply:
Why?

Reply:
But that is a different situation, you probably asked for the creation of a new dump while another one was being created. So it returns a `409 Conflict` which would be checked by the solution I proposed before. But ibdeed, you need to wait until the status of the last fumps is not `in_progress` (so done or failed) before creating a new one